dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -105,14 +110,14 @@ static StaticLinkageCheckOption parseArguments(String[] arguments) throws ParseE
     ImmutableList.Builder<String> mavenCoordinates = ImmutableList.builder();
     try {
       CommandLine commandLine = parser.parse(options, arguments);
-      if ((commandLine.hasOption("b") && commandLine.hasOption("c"))
-          || (commandLine.hasOption("c") && commandLine.hasOption("j"))
+      if ((commandLine.hasOption("b") && commandLine.hasOption("a"))
```

Comment:
How about something like 
```
if (Stream.of("b", "a", "j").filter(commandLine::hasOption).count() > 1) {
```

Better yet, you could build the `StaticLinkageCheckOption` object and then throw if more than one of the properties is not null or empty. Then you're not repeating the option letters again; and it's more meaningful a check.

Reply:
Applied the stream-filter-count. That's much cleaner.